apps/web/src/services/dashboards/pages/DashboardsMainPage.vue

```diff
@@ -119,14 +129,66 @@ const queryState = reactive({
 });
 
 /* Util */
-const getSearchedTreeData = (treeData: TreeNode<DashboardTreeDataType>[], searchFilters: ConsoleFilter[]): TreeNode<DashboardTreeDataType>[] => {
-    if (!searchFilters.length) return treeData;
-    return treeData.filter((node) => {
-        if (node.data.type === 'DASHBOARD') return true;
-        return !!node.children?.length;
+const getSearchedTreeData = (treeData: TreeNode<DashboardTreeDataType>[], searchedDashboardIdList: Set<string>): TreeNode<DashboardTreeDataType>[] => {
+    if (state.isSearching) return [];
```

Comment:
nitpick: I think, loading checking process is not the concern of this util function. I think it is necessary only in computed hooks!
Also, u commented tis function as 'util', so I think it must be pure. 
But this function checks state whose context is the component.

Before splitting the codes as util function, it would be helpful to check the list below:
- whether the function is pure
- whether the function has only one concern